src/platform/UM32/bus_quadspi_hal.c

```
@@ -0,0 +1,153 @@
+{
+
+}
+
+static const uint32_t quadSpi_addressSizeMap[] = {
+
+};
+
+static uint32_t quadSpi_addressSizeFromValue(uint8_t addressSize)
+{
+   
+}
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)